common/src/commonMain/sqldelight/io/xorum/codeforceswatcher/User.sq

```diff
@@ -0,0 +1,31 @@
+import kotlin.collections.List;
+
+CREATE TABLE User (
+    id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    avatar TEXT NOT NULL,
+    rank TEXT,
+    handle TEXT NOT NULL,
+    rating INTEGER,
+    maxRating INTEGER,
+    firstName TEXT,
+    lastName TEXT,
+    ratingChanges TEXT NOT NULL
+);
+
+getById:
+SELECT * FROM User WHERE id = ?;
+
+getAll:
+SELECT * FROM User;
+
+insert:
+INSERT OR REPLACE INTO User(avatar, rank, handle, rating, maxRating, firstName, lastName, ratingChanges) VALUES(?,?,?,?,?,?,?,?);
```

Comment:
I've see that you have `if` for 2 inserts, but they both are `INSERT OR REPLACE`, so not sure you can't get rid of one.

Reply:
The first one really insert User, second one - update. They are different, because second one have 'id' parametr, unlike first one.